tests/test_nautilus_librarian/utils.py

```diff
@@ -0,0 +1,41 @@
+import pathlib
+import subprocess  # nosec
+
+
+def debug_execute_console_command(multiline_command):
+    return execute_console_command(
+        multiline_command, print_output=True, print_command=True
+    )
+
+
+def execute_console_command(multiline_command, print_output=False, print_command=False):
```

Comment:
`print_output` and `print_command` are both unused.

Reply:
I use the function `debug_execute_console_command` whilst debugging a given test. I have to run a lot of console commands (git init, dvc add , ...) and it's pretty handy to print the full output to check what is going wrong. Maybe we can automatically change the behaviors of `debug_execute_console_command` when you run the tests with the option `pytest -s`, but I suppose you you do not always want to print all.

Reply:
However both arguments, `print_output` and `print_command`, are not referenced in the function body at all...